### PR TITLE
Replace method findOne by findOneBy

### PR DESCRIPTION
**Pull request by courentin**

### CODE REVIEW

File: Controller/SnapshotAdminController.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ public function createAction(Request $request = null)
         $snapshot = new $class();
 
         if ($request->getMethod() == 'GET' && $request->get('pageId')) {
-            $page = $pageManager->findOne(array('id' => $request->get('pageId')));
+            $page = $pageManager->findOneBy(array('id' => $request->get('pageId')));
```
Comment (Soullivaneuh) [inline Controller/SnapshotAdminController.php]: Why not `->find($request->get('pageId'))`?

**Comment by Soullivaneuh** [comment]
Could you please add some tests?

**Comment by core23** [comment]
I wonder why this could ever work... Looks like ``findOne`` never exists.

**Comment by Soullivaneuh** [comment]
According to the new [Sonata version management](https://github.com/sonata-project/SonataAdminBundle/issues/3053) and next major [release plan](https://github.com/sonata-project/SonataAdminBundle/issues/3731), this project has been refactored regarding branching and versioning.

If you see this message, your PR concerns a patch or a minor release and is not targeting the right branch.

So I'm closing this one, but don't see it as a refusal. If you think your work is still relevant and want to continue, feel free to reopen it on the right branch (e.g. the default one).

Regards.

**Pull request closed without merging**